doc/man3/EVP_VerifyInit.pod

```diff
@@ -49,7 +49,7 @@ implementation of digest I<type>.
 EVP_VerifyInit_ex() and EVP_VerifyUpdate() return 1 for success and 0 for
 failure.
 
-EVP_VerifyFinal()_with_libctx() and EVP_VerifyFinal() return 1 for a correct
+EVP_VerifyFinal()_ex() and EVP_VerifyFinal() return 1 for a correct
```

Comment:
The first `()` should be dropped -- existing error but might as well get it fixed.

Reply:
Typo fixed